target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-rax120.dts

```diff
@@ -0,0 +1,1010 @@
+			reg = <0>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			registers-number = <2>;
+			spi-max-frequency = <1000000>;
+			enable-gpios = <&tlmm 20 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led0 {
+			label = "Power";
+			gpios = <&led_gpio 0 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "led_wifi_son_orange";
+		};
+
+		led1 {
+			label = "24g_on";
+			gpios = <&led_gpio 1 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "led_wifi_son_orange";
+		};
+
+		led2 {
+			label = "5g_on";
+			gpios = <&led_gpio 2 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "led_wifi_son_blue";
+		};
+
+		led3 {
+			label = "USB2";
+			gpios = <&led_gpio 3 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0radio";
+		};
+
+		led4 {
+			label = "USB1";
+			gpios = <&led_gpio 4 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "led_5gl";
+		};
+
+		led5 {
+			label = "Internet";
+			gpios = <&led_gpio 5 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "led_5gh";
+		};
+
+		led6 {
+			label = "MultiGig:Green";
+			gpios = <&led_gpio 6 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "led_lan";
+		};	
+
+		led7 {
+			label = "MultiGig:Red";
+			gpios = <&led_gpio 10 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "led_lan";
+		};
+
+		led8 {
+			label = "MultiGig:White";
+			gpios = <&led_gpio 11 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "led_lan";
+		};
+
+		led9 {
+			label = "WPS";
+			gpios = <&tlmm 40 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "led_usb2_enable";
+		};
+	};
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+        #size-cells = <1>;
+		partition@0100000 {
+			label = "0:ART.bak";
+			reg = <0x01000000 0x00080000>;
+		};
+		partition@01080000 {
+			label = "config";
+			reg = <0x01080000 0x00100000>;
+		};
+		partition@01180000 {
+			label = "boarddata1";
+			reg = <0x01180000 0x00100000>;
+		};
+		partition@01280000 {
+			label = "boarddata2";
+			reg = <0x01280000 0x00100000>;
+		};
+		partition@01380000 {
+			label = "pot";
+			reg = <0x01380000 0x00100000>;
+		};
+		partition@01480000 {
+			label = "dnidata";
+			reg = <0x01480000 0x00500000>;
+		};
+		firmware@01980000 {
+			label = "firmware";
+			reg = <0x01980000 0x06400000>;
+		
+			partition@1 {
+				label = "kernel";
+				reg = <0x01980000 0x00620000>;
+			};
+			partition@2 {
+				label = "rootfs";
+				reg = <0x01fa0000 0x05de0000>;
+			};
+		};
+		partition@07d80000 {
+			label = "language";
+			reg = <0x07d80000 0x00380000>;
+		};
+		partition@08100000 {
+			label = "cert";
+			reg = <0x08100000 0x00100000>;
+		};
+		partition@08200000 {
+			label = "mtdoops";
+			reg = <0x08200000 0x00080000>;
+		};
+		partition@08280000 {
+			label = "router_analytics";
+			reg = <0x08280000 0x00200000>;
+		};
+		partition@08480000 {
+			label = "ntgrdata";
+			reg = <0x08480000 0x06400000>;
+		};
+		partition@0E880000 {
+			label = "reserved";
+			reg = <0x0E880000 0x11780000>;
+		};
+	};
+};
+
+&clocks {
+	sleep_clk: sleep_clk {
```

Comment:
This is a SoC clock, already described in the SoC DTSI, remove it.

Reply:
I defined this node due to a variation in the clock-frequency. Seems to be working fine after removal. 

sleep_clk clock-frequency = <32000> (OEM) vs <32768> (ipq8074.dtsi)

Reply:
It is for 5.15 but it's been fixed upstream since. Removal will allow this fix to eventually apply. 
https://github.com/torvalds/linux/commit/f607dd767f5d6800ffbdce5b99ba81763b023781

Reply:
Ahh, it was a late addon to 5.19 PR, that's why I haven't backported it yet.
32.768 kHz is the correct value, I will backport the patch.
Even with 32.0 kHz it only matters if the frequency is used as the basis for calculating one of the clocks that use the sleep one as source clock (I think only USB sleep clocks do this)